src/Examples/docs_examples.jl

```diff
@@ -40,11 +40,16 @@ function draw_zoomlenses(filenames::Vector{<:Union{Nothing,AbstractString}} = re
     directions = Directions.Constant(0.0, 0.0, -1.0)
     raygenerator = Sources.Source(; transform, origins, directions)
 
+    aspherics = [
+        [0, 0, 0, 1.0386E-04, 0, 1.4209E-07, 0, -8.8495E-09, 0, 1.2477E-10, 0, -1.0367E-12, 0, 3.6556E-15],
+        [0, 0, 0, 4.2721E-05, 0, 1.2484E-07, 0, 9.7079E-09, 0, -1.8444E-10, 0, 1.8644E-12, 0, -7.7975E-15],
+        [0, 0, 0, 1.1339E-04, 0, 4.8165E-07, 0, 1.8778E-08, 0, -5.7571E-10, 0, 8.9994E-12, 0, -4.6768E-14],
+    ]
     syss = [
         AxisymmetricOpticalSystem{Float64}(DataFrame(
             SurfaceType = ["Object", "Stop", "Standard", "Standard", "Standard", "Aspheric", "Standard", "Standard", "Aspheric", "Aspheric", "Standard", "Standard", "Standard", "Standard", "Standard", "Standard", "Image"],
             Radius = [Inf64, Inf64, -1.6202203499676E+01, -4.8875855327468E+01, 1.5666614444619E+01, -4.2955326460481E+01, 1.0869565217391E+02, 2.3623907394283E+01, -1.6059097478722E+01, -4.2553191489362E+02, -3.5435861091425E+01, -1.4146272457208E+01, -2.5125628140704E+02, -2.2502250225023E+01, -1.0583130489999E+01, -4.4444444444444E+01, Inf64],
-            Parameters = [missing, missing, missing, missing, missing, [(4, 1.0386E-04), (6, 1.4209E-07), (8, -8.8495E-09), (10, 1.2477E-10), (12, -1.0367E-12), (14, 3.6556E-15)], missing, missing, [(4, 4.2721E-05), (6, 1.2484E-07), (8, 9.7079E-09), (10, -1.8444E-10), (12, 1.8644E-12), (14, -7.7975E-15)], [(4, 1.1339E-04), (6, 4.8165E-07), (8, 1.8778E-08), (10, -5.7571E-10), (12, 8.9994E-12), (14, -4.6768E-14)], missing, missing, missing, missing, missing, missing, missing],
```

Comment:
Not a big fan of this, firstly we only support even aspherics so either we need to double the index, or every other index will always be 0. Secondaly, it is might be the case that only high order aspherics are used so for a single parameter (12th aspheric term) we need a list of length 24. The same arguments are true for most surface types with many parameters eg zernike, qtype, chebyshev. This gets even worse where we have multiple parameters eg zernike surface which has both zernike terms and aspherics, how do we input these in a flat list format?

Is it not possible to support arbitrarily typed parameters, or even a dictionary of param_name: param_value so that not all parameters need to be provided for a surface type which has multiple parameters. This makes input _much_ clearer to the user as well, otherwise they have to somehow parse a flat list which might potentially represent a number of different parameters